TYPO3.Media/Resources/Private/Templates/Asset/Edit.html

```diff
@@ -107,5 +107,5 @@
 </f:section>
 
 <f:section name="Scripts">
-	<script type="text/javascript" src="{f:uri.resource(package: 'TYPO3.Media', path: 'Scripts/edit.js')}"></script>
+	<script type="text/javascript" src="{f:uri.resource(package: 'TYPO3.Neos', path: 'Scripts/edit.js')}"></script>
```

Comment:
Sorry, but is that good idea? I may not be up to speed when it comes to (de-)coupling Media and Neos, but this looks wrong on many levels…